foundation/foundation-upgrade/src/main/resources/migrations.json

```
@@ -188,6 +188,26 @@
             "includes": ["pom.xml"]
           }
         ]
+      },
+      {
+        "name": "ml-train-pipeline-docker-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.version_specific.MlTrainPipelineDockerMigration",
+        "fileSets": [
+          {
+            "includes": ["pom.xml"]
+          }
+        ]
+      },
+      {
+        "name": "ml-train-image-tag-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.version_specific.MlTrainImageTagMigration",
+        "fileSets": [
+          {
+            "includes": [
+              "*-deploy/src/main/resources/apps/model-training-api/values-dev.yaml"
```

q: Is there a concern that the downstream project has changed the helm chart dir from model-training-api?

---

i don't think it's an immediate concern, it seems unlikely that a user would change the directory name given it's auto-generated with the pipeline step

---

Easy enough to check the sibling _Chart.yaml_ for `aissemble-model-training-api-chart` and not have to worry at all. I'd also recommend dropping the wildcard module match and let this migration run from the module itself instead of root:
```suggestion
              "src/main/resources/apps/model-training-api/values-dev.yaml"
```

---

removed migration altogether